diceRoll/diceRollUnitTests.js

```diff
@@ -46,14 +153,15 @@ function UnitTest(testName, functionToBeTested) {
   this.functionToBeTested = functionToBeTested;
   this.report;
   try {
-    if (!functionToBeTested()) {
```

Comment:
REVERTED, because this was approving functions that returned a string, treating it as a `true` statement